src/libraries/ErrorsLib.sol

```diff
@@ -26,4 +26,10 @@ library ErrorsLib {
 
     /// @notice Thrown when rewards have already been claimed.
     string internal constant ALREADY_CLAIMED = "already claimed";
+
+    /// @notice Thrown when the value is already set.
+    string internal constant ALREADY_SET = "already set";
+
+    /// @notice Thrown when the submitted pending root is the same as the current one.
+    string internal constant ROOT_ALREADY_SET = "root already set";
```

Comment:
This is not really true, I would simplify things and just use `ALREADY_SET` above as well for the root

Reply:
But Ive written in the PR desc why Ive defined 2 errors. Have you argument against using 2 errors?

Reply:
Why would we want a specific error for the root ?

Reply:
Because when submitting a pending root, we check if the value is not equal to the pending one or the current one. We want to throw 2 different errors for 2 different cases